test/others/criu-ns/run.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+
+import multiprocessing
+import os
+import pty
+import shutil
+import subprocess
+import sys
+import time
+
+
+CRIU_BIN = "../../../criu/criu"
+CRIU_NS = "../../../scripts/criu-ns"
+
+
+def check_dumpdir():
+    if os.path.isdir("dumpdir"):
+        shutil.rmtree("dumpdir")
+    os.mkdir("dumpdir", 0o755)
```

Comment:
It might be good to store the value `"dumpdir"` in constant and use this constants instead.
For example:

```Python
IMG_DIR = "dumpdir"
...
os.mkdir(IMG_DIR, 0o755)

Reply:
Yeah, make sense, will do.

Reply:
done.